internal/controllers/machine/machine_controller.go

```diff
@@ -687,6 +685,15 @@ func (r *Reconciler) shouldAdopt(m *clusterv1.Machine) bool {
 }
 
 func (r *Reconciler) watchClusterNodes(ctx context.Context, cluster *clusterv1.Cluster) error {
+	log := ctrl.LoggerFrom(ctx)
+
+	if cluster.GetConditions() == nil || conditions.IsFalse(cluster, clusterv1.ControlPlaneInitializedCondition) {
+		log.V(5).Info("Skipping node watching setup because control plane is not initialized", "cluster", cluster.Name)
+		return nil
+	}
+
+	log.V(5).Info("Watching cluster nodes", "cluster", cluster.Name)
```

Comment:
Do we need this? It would permanently log noise after ControlPlaneInitializedCondition goes true.

Reply:
it's a verbosity 5 log, if it's used for debugging then I guess it can be a bit noisy

Reply:
@enxebre We are working on something (in the context of the other recent log changes like JSON logging, upstream CR adjustments). Just didn't have the time yet to finalize it a bit more and publish it

Reply:
+1 to remove this log line specifically

Reply:
@alexander-demichev Do you have time to fix this comment?

Reply:
hey, I addressed the comment :)